project.godot

```diff
@@ -13,6 +13,25 @@ config_version=3
 config/name="Random Generic Platformer"
 run/main_scene="res://scenes/StartMenu.tscn"
 
+[input]
```

Comment:
Afaik godot always only saves settings different from default here, that's why this appeared.